packages/fast-components-react-base/src/button/button.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 import { get } from "lodash-es";
-import Foundation, { HandledProps } from "../foundation";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
```

Comment:
This is not a component library as such, it has one default export the `Foundation` component (as indicated by the name of the package like you mentioned), it may have secondary exports but this is the purpose of the package. We do this in other packages where there is an expected default export such as `Form` for the form generator.

Yes, this is going into the breaking changes releases branch.